heka/meta/heka.yml

```diff
@@ -360,11 +361,51 @@ aggregator:
           threshold: 1
     - status: warning
       trigger:
+        logical_operator: and
+        rules:
```

Comment:
Like it was done in nova formula for consistency. A count of instances up >= 2 while others are down or critical is warning.